CONTRIBUTING.md

```diff
@@ -7,5 +7,7 @@ run the hooks:
 
 ``` console
 $ pre-commit install
-$ pre-commit run --all-hooks
+# do a test run on all the files in the repo
+$ pre-commit run -a
```

Comment:
suggestion: long-opt for readability and search-ability (pretty sure this is the arg I meant to add originally :facepalm:)

```suggestion
$ pre-commit run --all-files
```